sources/pmm.2.27.0.pmm-server.json

```diff
@@ -42,6 +42,12 @@
             "image_hash": "1b6b5dda3c9ec50128f8e4fd0a9e07feb16bb262b92f8d31f09fe19df51b069a",
             "status": "recommended",
```

Comment:
Can two versions be recommended at the same time?

Reply:
Fixed